src/components/Header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react"
+
+export default function Header() {
+    const [unpaid, setUnpaid] = useState(0)
+    useEffect(() => {
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
```

Comment:
Having a duplicate fetch request here feels wrong to me, however I do think it's actually unavoidable, since you'll be displaying this header on pages that otherwise don't need to get invoices.

However, you can reduce the amount of data being fetched, and the work you have to do, by adding ?status=2 onto the end of the url (https://invoicing-api.dev.io-academy.uk/invoices?status=2). That way the API will only give you pending invoices, and you can just use .length to get the count.